example_rules/example_single_metric_agg.yaml

```diff
@@ -0,0 +1,35 @@
+name: Metricbeat CPU Spike Rule
+type: metric_aggregation
+
+es_host: localhost
+es_port: 9200
+
+index: metricbeat-*
+
+buffer_time:
+  hours: 1
+
+metric_agg_key: system.cpu.user.pct
```

Comment:
will this just work like this? I didn't see the use of `lookup_es_key` in this PR, and I thought that we needed special handling for nested fields?